sdk/communication/azure-communication-administration/README.md

```diff
@@ -2,8 +2,8 @@
 
 Azure Communication Administration is used for managing users and tokens for Azure Communication Services.
 
-<!-- [Source code][source] | [Package (Maven)][package] | [API reference documentation][api_documentation]
-| [Product documentation][azconfig_docs] -->
+[Source code][source] | [Package (Maven)][package] | [API reference documentation][api_documentation]
+| [Product documentation][azconfig_docs]
```

Comment:
I think we need to replace all azconfig_docs / appconfig links with links to acs